addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -324,7 +320,11 @@ def from_html(self, model, field, element):
 
         # parse from string to datetime
         lg = self.env['res.lang']._lang_get(self.env.user.lang) or get_lang(self.env)
-        dt = datetime.strptime(value, '%s %s' % (lg.date_format, lg.time_format))
+        try:
+            datetime_format = f'{lg.date_format} {lg.time_format}'
+            dt = datetime.strptime(value, datetime_format)
+        except ValueError:
+            raise ValidationError(_("The datetime %s does not match the format %s", value, datetime_format))
```

Comment:
Are you sure this is needed?
We made a generic fix to catch those errors and display it in a non blocking way with https://github.com/odoo/odoo/commit/119ad2fb7b666ca7e4a89c8d9fd5a1427308657f#diff-3fbc23861d612695a09633388940ac2c62aa2dadba54d4f55b29ba9fe72be70cR44-R48
(Which was replacing this kind of specific fix)

Error in `from_html` are not supposed to be blocking anymore (as the task seems to suggest, but it was created ~ the same time as the commit was merged so maybe not needed anymore)

Reply:
Hello
Indeed, it's no longer needed (I will update the commit message). But it could still be a good idea to have this one for the datetime. Since the user needs to respect a specific format that can change based on the lang. What do you think ? If in your opinion it's overkill, I can indeed remove this part